adafruit_motorkit.py

```diff
@@ -50,8 +58,12 @@ class MotorKit:
     Alternately, if using with multiple I2C devices, you can specify the I2C bus."""
 
     def __init__(
-        self, address=0x60, i2c=None, steppers_microsteps=16, pwm_frequency=1600
-    ):
+        self,
+        address: int = 0x60,
+        i2c: Optional[I2C] = None,
+        steppers_microsteps: int = 16,
+        pwm_frequency: float = 1600,
```

Comment:
Should the default value here be `1600.0`? Or should the type be int? It's listed as float as well inside of PCA9685 library so I'm guessing that is correct and that we'd want 1600.0 here to match.